float64 time
int32 nr_batch
int32 nr_persons

float32[] colour_vectors
int32[] x_positions
int32[] y_positions
int32[] z_positions